tests/importance_tests/test_init.py

```diff
@@ -103,15 +107,19 @@ def objective(trial: Trial) -> float:
 
         # Sanity check for param importances
         assert all(0 <= x < float("inf") for x in param_importance.values())
+        if normalize:
+            assert len(param_importance) == 0 or np.isclose(sum(param_importance.values()), 1.0)
```

Comment:
I think the first condition is always false in this test function.
```suggestion
            assert np.isclose(sum(param_importance.values()), 1.0)
```